server/call/session.py

```diff
@@ -135,13 +140,37 @@ def get_participant_count(self) -> int:
         presence = res.json()
         return presence['total_count']
 
+    def get_room(self, room_url: str) -> Room:
+        """Retrieves a Daily room on the configured domain."""
+        if not room_url:
+            raise Exception("Room URL must be provided to retrieve a room")
+        room_name = os.path.basename(room_url)
+        url = f'{self._config.daily_api_url}/rooms/{room_name}'
+        headers = self.get_auth_headers()
+
+        res = requests.get(url,  headers=headers)
+
+        if not res.ok:
+            raise Exception(
+                f'Failed to get room {res.status_code}'
```

Comment:
Would more than just the status code be helpful, too, to return?

Reply:
Good point, added body